Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+
+**Model outputs**
+
+These are usually reproducible, even if it might not be possible to get bitwise reproducibility if underlying libraries change or the machine you have used is retired. 
+Such a decision should be weighed against the cost, in money and time, of reproducing your work.
+On the other side storing the data can also be expensive, as the model output are usually large,  and of little use if not done properly.
+That is where you need to discuss with your supervisor, and possibly other collaborators, as the answer might vary depending on how likely someone else might find this data useful.
+As model output includes raw output, restart files and post-processed output, they should all be treated differently. usually if keeping raw output this is stored on tape, it is a bit harder to retrieve but tape is less expensive than other storage. Only some of the restart files are kept (if any). Ideally, post-processed should be published if it has been used for publication or if it's going to be shared with others. 
+
+**Output of lengthy processing**
+
+ It might feel necessary to keep those files but we would argue it is usually not worth the cost of the storage unless there is a clear indication they will be used again soon. The first reason is that with current modern programming techniques a lot of lengthy processing can be shortened significantly. The second reason is the very real cost of the storage is not worth the very hypothetical time saved in the future.
+
+
+## Publishing data and code
+
+You should publish your files if the files are frequently used by several other persons. Ideally, that data should be published as soon as its usefulness to others is clear, not just when you leave.
+
+If your files are at NCI, that is all you need to do. If your files are held at your institution, you may need to make sure there is a copy that is accessible by others and owned by someone who is likely to stay for years to come. This might depend on your institution data services.
+
+**Sharing files without publication**
+
+Some files might be useful to a small cluster of people but not be worth publishing. In that case, you simply need to change the ownership of the files: get the files to be owned by someone who stays after you.
+
+If the files are at NCI, you can not change the files' ownership. You and the Lead CI of the project owning the files will need to contact help@nci.org.au so NCI staff can do it for you.
+
+```{warning}
+Often researchers shared data with others without publishing it even when this data is used for new publication, this should be avoided at any cost. It does create an issue for the creator of the data who won't get recognition and for the user who might struggle to refer to the exact data when publishing a paper. As it is now a requirement to publish data associated with a paper it is always bets to publish before sharing!
```

Comment:
```suggestion
In the past, researchers often shared data with others before publishing it, even when this data was used for a new publication. This practice should be avoided at any cost. It creates an issue for the creator of the data who won't get recognition, and for the user who might struggle to refer to the exact data when publishing a paper. As it is now required to publish data associated with a paper, it is always best to publish before sharing!
```

Reply:
I might change the verb tense at the start as this happens all the time not just in the past